tests/test_lens.py

```diff
@@ -0,0 +1,66 @@
+from pathlib import Path
```

Comment:
tests should be in their own directory. So put the lens test in /tests/lens, and the soap tests in tests/soap/